src/app/content/PrivateSidebar.js

```diff
@@ -21,15 +26,40 @@ const PrivateSidebar = ({ user, contactGroups }) => {
             },
             text: c('Link').t`Contacts`,
             link: '/contacts'
+        },
+        !loadingUserKeys && {
+            type: 'button',
+            className: 'alignleft',
+            icon: 'import',
+            text: c('Link').t`Import`,
+            onClick() {
+                createModal(<ImportModal userKeysList={userKeysList} />);
+            }
+        },
+        !loadingUserKeys && {
+            type: 'button',
+            className: 'alignleft',
+            icon: 'export',
+            text: c('Link').t`Export`,
+            onClick() {
+                createModal(<ExportModal userKeysList={userKeysList} />);
```

Comment:
This always does an "export all". In the case a contact is selected, would we want to export just one?